lib/libcxxrt/Version.map

```diff
@@ -250,7 +250,8 @@ CXXABI_1.3.9 {
 } CXXABI_1.3.6;
 
 CXXABI_1.3.11 {
-    __cxa_init_primary_exception;
+# Not present in CHERI libcxxrt
```

Comment:
I think it might be better to update libcxxrt, these kinds of diffs are ready to forget about.

Reply:
We'd need to decide how to do the update.  It didn't quite look trivial to me as I think there are many local diffs in our libcxxrt we could now drop (MIPS support, support for building without unwinding or RTTI).  I had kind of punted and decided to just patch this for now knowing we'd have to fix this up when we do update libcxxrt.  libcompiler_rt is in a similar boat except we do actually update that periodically when doing LLVM merges.